### DISCUSSION TITLE
stat-computing-r-python/part-tools/03-Rstudio-interface

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#19] [General]
# stat-computing-r-python/part-tools/03-Rstudio-interface

https://srvanderplas.github.io/stat-computing-r-python/part-tools/03-Rstudio-interface.html

**Comment by dsander04** [comment]
Tiny typo in 3.4! "Can" is spelled "cna" in the third to last line

**Reply by srvanderplas** [reply]
Just pushed a fix! Thanks!
